### PR TITLE
new animax source

### PR DESCRIPTION
**Pull request by jlapitan**

### CODE REVIEW

File: channels/kr.m3u [hunk 1]
```diff
@@ -165,3 +165,5 @@ http://54.180.143.6:9981/stream/channelid/750850617
 http://54.180.143.6:9981/stream/channelid/1805607194
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/r1WTYTa.png" group-title="",현대홈쇼핑+샵 / + Shop
 http://54.180.143.6:9981/stream/channelid/965307250
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
+http://211.49.191.148:9981/stream/channelid/1158483964
```
Comment (boomski) [inline channels/kr.m3u]: afbeelding
Link does not work

File: channels/kr.m3u [hunk 2]
```diff
@@ -167,3 +167,5 @@ http://54.180.143.6:9981/stream/channelid/1805607194
 http://54.180.143.6:9981/stream/channelid/965307250
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
 http://211.49.191.148:9981/stream/channelid/1158483964
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/WRzx3F9.png" group-title="",Fox Life
+http://211.49.191.148:9981/stream/channelid/1960414728
```
Comment (boomski) [inline channels/kr.m3u]: afbeelding
Link does not work

File: channels/kr.m3u (hunk 1 above)
Comment (jlapitan) [reply channels/kr.m3u]: Screen Shot 2019-10-15 at 1 43 28 AM

File: channels/kr.m3u (hunk 2 above)
Comment (jlapitan) [reply channels/kr.m3u]: Screen Shot 2019-10-15 at 1 45 43 AM
link is working

**Comment by boomski** [comment]
I still getting error on these links  , maybe geoblocked?

**Comment by jlapitan** [comment]
> I still getting error on these links , maybe geoblocked?
Not sure but these are the results test link

https://streamtest.in/logs/http-211-49-191-148-9981-stream-channelid-1960414728-696b17ef-dfa1-4465-a69c-4cdc8f701ade

https://streamtest.in/logs/http-211-49-191-148-9981-stream-channelid-1158483964-3a125a72-cdd9-450a-ab72-d2bfa100019e

File: channels/kr.m3u (hunk 1 above)
Comment (freearhey) [reply channels/kr.m3u]: stream works for me. with lags but works (tested with VLC and streamtest)

File: channels/kr.m3u (hunk 2 above)
Comment (freearhey) [reply channels/kr.m3u]: same thing, stream works but it is not possible to watch due to lags

File: channels/qa.m3u [hunk 3]
```diff
@@ -17,3 +17,6 @@ https://alrayyan1live.azureedge.net/alrayyan1/alrayyan1.smil/chunklist_b1428000.
 https://alrayyan2live.azureedge.net/alrayyan2/alrayyan2.smil/chunklist_b1428000.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
 http://95.170.215.101/hls/m3u8/Bein-Sport-HD1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```
Comment (freearhey) [inline channels/qa.m3u]: currently not working (tested with VLC and streamtest)

**Review by freearhey — CHANGES_REQUESTED** [review]
Please remove all broken links from PR so that we can accept it.

File: channels/qa.m3u (hunk 3 above)
Comment (jlapitan) [reply channels/qa.m3u]: non-working stream has been removed

**Review by freearhey — APPROVED** [review]

**Pull request merged**